src/components/app/app.jsx

```diff
@@ -1,11 +1,21 @@
 import React from "react";
-import Main from '../main/main.jsx';
+import Main from "../main/main.jsx";
+import PropTypes from "prop-types";
 
 const App = (props) => {
-  // eslint-disable-next-line react/prop-types
-  const {numberOfRent} = props;
+  const {placeCards} = props;
```

Comment:
```suggestion
  const {places} = props;
```
Старайся выбирать названия очень тщательно.
В данном случае это места или предложения.
Карточки - это только при отображении